app/models/client.rb

```diff
@@ -0,0 +1,130 @@
+class Client
+
+  # Intercom API Client
+  def initialize(organization)
+    @intercom_client = Intercom::Client.new(app_id: ENV['intercom_api_id'], api_key: ENV['intercom_api_key'])
+  end
+
+  def find_all(entity, created_since = nil)
+    method = "get_#{entity.pluralize}"
+    created_since = (Date.today - Date.parse(created_since)).to_i
+    send(method, created_since)
+  end
+
+  def get_users(created_since = nil)
+    users = if created_since
+              @intercom_client.users.find_all(created_since: created_since)
+            else
+              @intercom_client.users.all
+            end
+
+    entities = users.collect do |user|
```

Comment:
Isn't there a way to transform a user into a hash directly (methods like `to_hash`, `attributes`, ...)?